doc/act_45602_00_000.md

```diff
@@ -0,0 +1,236 @@
+
+# 45602 Lamp Module with Dimmer Control
+
+This describes the Z-Wave device **45602**, manufactured by **ACT - Advanced Control Technologies** with the thing type UID of ```act_45602_00_000```. 
+
+Lamp Module with Dimmer Control
+
+## Channels
+The following table summarises the channels available for the 45602 Lamp Module with Dimmer Control.
+
+| Channel | Channel Id | Channel Type UID | Category | Item Type |
+|---------|------------|------------------|----------|-----------|
+| Dimmer | switch_dimmer | switch_dimmer | DimmableLight | Dimmer |
+
+
+
+### Dimmer
+
+#### Restore Last Value
+
+Restore Last Dimming level on ON.
+
+
+| Property         | Value    |
+|------------------|----------|
+| Configuration ID | config_restoreLastValue |
+| Data Type        | BOOLEAN || Default Value | true |
+| Options | Restore Last Value (true) |
+|  | Restore Full Brightness (false) |
+
+
+
+
+
+
+### Device Configuration
+The following table provides a summary of the configuration parameters available in the 45602 Lamp Module with Dimmer Control.
+Detailed information on each parameter can be found below.
+
+| Parameter   | Description |
+|-------------|-------------|
+| 5: Ignore Start Level When Receiving Dim Commands |  |
+| 7: Dim Rate Step Count - ZWave Control | Number of steps the dimmer will change when controlling via ZWave |
+| 8: Dim Rate Step Duration - ZWave Control | Timing of each step the dimmer will change when controlling via ZWave |
+| 9: Dim Rate Step Count - Manual Control | Number of steps the dimmer will change when controlling manually |
+| 10: Dim Rate Step Duration - manual Control | Timing of each step the dimmer will change when controlling manually |
+| 11: Dim Rate Step Count - ZWave All on/off | Number of steps the dimmer will change when receiving a ZWave all on/off command |
+| 12: Dim Rate Step Duration - ZWave All on/off | Timing of each step the dimmer will change when receiving a ZWave all on/off command |
+| 29: Load Sensing | Sense when existing switch is used to turn on the lamp |
+
+
+
+
+#### 5: Ignore Start Level When Receiving Dim Commands
+
+  
+
+
+# Overview #
```

Comment:
I get where these come from but they do not make any sense in the documents created. Any quick way to remove them?

Reply:
I'm not sure I understand your point - what do you want removed? The whole heading, or the fact that it's an H1 heading or something else?

Reply:
The headings are probably okay but they are H1 making an article like this one hard to read/ make sense of: https://github.com/cdjackson/org.openhab.binding.zwave/blob/638bb9a8c6ed38e635a275c303ec4c1190fd18cd/doc/benext_alarmsound_00_000.md

Reply:
Agreed, but I'm not sure if I can easily change that. Please remember that the main purpose of the files is the XML thing descriptions and I'm trying to provide documentation from that. Within the context of stand alone files, H1 makes sense I think, but when they are inserted into another structure, it's not so good. Maybe I can find a way to change all headings "down" a few steps, but I'm not so sure - I'll see what I can do...

Reply:
I've managed to code up something to increase the header indentation.